tensorboard/webapp/metrics/views/right_pane/scalar_column_editor/scalar_column_editor_container.ts

```diff
@@ -54,4 +55,14 @@ export class ScalarColumnEditorContainer {
   }) {
     this.store.dispatch(dataTableColumnToggled({dataTableMode, headerType}));
   }
+
+  onScalarTableColumnEdit({
+    dataTableMode,
+    headers,
+  }: {
+    dataTableMode: DataTableMode;
+    headers: ColumnHeader[];
+  }) {
+    this.store.dispatch(dataTableColumnEdited({dataTableMode, headers}));
```

Comment:
Would you mind extracting the inline type from `dataTableColumnEdited` then importing and using it here?
Technically this could be extracted using just TS but I don't think it reads that well
`Parameters<typeof dataTableColumnEdited>[0]`

Bonus points for doing the same cleanup on line 56.